multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/ProcessLoggerProvider.java

```diff
@@ -62,30 +66,84 @@ private String getTaskId(DelegateExecution execution) {
     }
 
     private ProcessLogger createProcessLogger(String spaceId, String correlationId, String activityId, String loggerName, String logName,
-                                              PatternLayout layout) {
-        Logger logger = Logger.getLogger(loggerName);
-        File logFile = getLocalFile(loggerName);
-        logger.removeAllAppenders();
-        logger.addAppender(createAppender(logger.getLevel(), logFile, layout));
-        return new ProcessLogger(logger, logFile, logName, spaceId, correlationId, activityId);
+                                              AbstractStringLayout abstractStringLayout) {
+        File logFile = getLocalFileByLoggerName(loggerName);
+        LoggerContext loggerContext = initializeLoggerContext(loggerName, logFile, abstractStringLayout);
+        Logger logger = loggerContext.getLogger(loggerName);
+        return new ProcessLogger(loggerContext, logger, logFile, logName, spaceId, correlationId, activityId);
     }
 
-    protected File getLocalFile(String loggerName) {
+    protected File getLocalFileByLoggerName(String loggerName) {
         String fileName = loggerName + LOG_FILE_EXTENSION;
         return new File(DEFAULT_LOG_DIR, fileName);
     }
 
-    private Appender createAppender(Level level, File logFile, PatternLayout layout) {
-        FileAppender appender = new FileAppender();
+    private LoggerContext initializeLoggerContext(String loggerName, File logFile, AbstractStringLayout abstractStringLayout) {
+
+        LoggerContext loggerContext = new LoggerContext(loggerName);
+        FileAppender fileAppender = initializeStartedAppender(loggerContext, logFile, abstractStringLayout, loggerName);
+        fileAppender.start();
```

Comment:
What does it mean start() ?

Reply:
the .start() method is required in order to make the Filter available for use.